platform/security/idp/security-idp-server/src/main/java/org/codice/ddf/security/idp/server/IdpEndpoint.java

```diff
@@ -1200,10 +1263,11 @@ private Response continueLogout(
             logoutState.isPartialLogout() ? StatusCode.PARTIAL_LOGOUT : StatusCode.SUCCESS;
         logoutObject =
             logoutMessage.buildLogoutResponse(
-                SystemBaseUrl.constructUrl("/idp/logout", true),
+                SystemBaseUrl.constructUrl(IDP_LOGOUT, true),
                 status,
                 logoutState.getOriginalRequestId());
         relay = logoutState.getInitialRelayState();
+        logoutStates.decode(cookie.getValue(), true);
```

Comment:
This line was the sole merge conflict between my cherrypick from 2.11.x and master. In master, it had been deleted; in my branch, I had removed the unnecessary assignment of the return value from the method. It should remain, as `decode()` has a side-effect.